incubator/hnc/docs/user-guide/concepts.md

```diff
@@ -455,5 +456,108 @@ problem, and will generally require human intervention to resolve.
 
 > _Note: HNC v0.5 reported issues with objects as part of the non-standard
 > condition schema. These have been removed and replaced by standard Events in
-> HNC v0.6 since events are more standard, scalable and loggable.
+> HNC v0.6 since events are more standard, scalable and loggable._
+
+<a name="admin-labels">
+
+### Labels and annotations
```

Comment:
Maybe also categorize the labels/annotations into two:
1) Added by HNC (most of them)
2) Added by users (exceptions)

Reply:
Done